src/library_gl.js

```diff
@@ -728,7 +728,11 @@ var LibraryGL = {
         ret = allocate(intArrayFromString(gl_exts.join(' ')), 'i8', ALLOC_NORMAL);
         break;
       case 0x8B8C /* GL_SHADING_LANGUAGE_VERSION */:
+#if USE_WEBGL2
+        ret = allocate(intArrayFromString('OpenGL ES GLSL 3.00 (WebGL2)'), 'i8', ALLOC_NORMAL);
```

Comment:
Yeah I was wondering why it was hardcoded. Should I just close this PR and let you handle it?